lib/scorecard.rb

```diff
@@ -0,0 +1,54 @@
+class Scorecard
+  attr_reader :scorecard, :roll, :frame_number
+
+  def initialize
+    @roll = 1
+    @scorecard = [{}]
+    @frame_number = 1
+  end
+
+  def add_score(score)
+    if @roll == 1
+      if @scorecard[@frame_number - 1][:strike?] == true || @scorecard[@frame_number - 1][:spare?] == true
```

Comment:
not 100% sure but when using || you don't need to state that they == true. the conditions checks for a 'true' evaluation